IMPLEMENTATION.md

```diff
@@ -50,3 +50,55 @@ The `ValidationIssue` class and its types:
   - `ContentValidationIssues.ts`: Issues that are found in tile content or external tilesets
 - For validation issues that refer to the tile content, each `ValidationIssue` can have an array of `causes`. This can be filled, for example, with the information from the glTF validator that caused the validation to fail
 
+## API Definition
+
+The API definition is tracked with https://api-extractor.com
+
+- Install api-extractor:
+ 
+  `npm install -g @microsoft/api-extractor`
+
+- Install api-documenter:
+    
+  `npm install -g @microsoft/api-documenter`
+
+- Run the TypeScript compiler to generate the build output:
+
+  `npm run build`
+
+- Invoke api-extractor:  
+
+  `api-extractor run --config api-extractor.jsonc --local --verbose`
```

Comment:
This line fails for me on Windows / Git Bash if `etc` doesn't already exist:

```
Error: Unable to create the API report file. Please make sure the target folder exists:
C:\Users\slilley\Code\3d-tiles-validator\etc
```

Reply:
Done by updating the `npm run docs` script. But... when running it for the first time, it complains
```
Warning: The API report file was missing, so a new file was created. Please add this file to Git:
C:\3d-tiles-validator\etc\3d-tiles-validator.api.md
```
so it appears this file should be committed to git. This makes sense from the perspective of "tracking" the API and possible changes. But I find it a bit unfortunate. Clients might not be aware of the use of this file, and I hope that it does not cause them to assume that this is the API documentation or so...